storybook/test/index.js

```diff
@@ -6,6 +6,20 @@ import initStoryshots, {
 } from '@storybook/addon-storyshots';
 import path from 'path';
 
+let MutationObserver;
```

Comment:
Can you add a comment referencing the PR I opened to update `jest` and `jsdom`?

Reply:
Added, thanks for the review